project/src/components/app/app.tsx

```diff
@@ -1,5 +1,13 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import Main from '../../pages/main/main';
+
+type AppScreenProps = {
+  placesToStay: number;
 }
 
-export default App;
+export default function App({placesToStay}: AppScreenProps): JSX.Element {
+  return (
+    <Main
+      placesToStay = {placesToStay}
+    />
```

Comment:
Когда пропсов <= 3-4 то лучше располагать в строку, пробелы вокруг равно не нужны
```suggestion
    <Main placesToStay={placesToStay} />
```